### PR TITLE
Misc. Aetheria updates

### PR DESCRIPTION
**Pull request by LtRipley36706**

### CODE REVIEW

**Comment by LtRipley36706** [comment]
more work to do on a few edge case areas

File: Source/ACE.Server/Entity/Aetheria.cs [hunk 1]
```diff
@@ -133,13 +142,18 @@ public static void UseObjectOnTarget(Player player, WorldObject source, WorldObj
 
             actionChain.AddAction(player, () => ActivateSigil(player, source, target));
 
+            player.EnqueueMotion(actionChain, MotionCommand.Ready);
```
Comment (gmriggs) [inline Source/ACE.Server/Entity/Aetheria.cs]: player.IsBusy = false should be after the return to ready state animation

IsBusy=true/false should also be kept together in the same function for consistency and clarity

**Comment by LtRipley36706** [comment]
think this is in a good state now for aetheria improving support

File: Source/ACE.Server/WorldObjects/Hotspot.cs [hunk 2]
```diff
@@ -152,32 +152,35 @@ private void Activate(Player player)
             var amount = DamageNext;
             var iAmount = (int)Math.Round(amount);
 
+            var pAmount = 0;
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/Hotspot.cs]: are 3 amounts really needed here?

the whole point of iAmount was to track the integer amount through the pipeline, for the final output message

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/Hotspot.cs]: iAmount wasn't changing on 3 of the 4, i can make it do that of course, but at the same time i didn't see the harm either way

File: Source/ACE.Server/WorldObjects/Hotspot.cs [hunk 3]
```diff
@@ -152,32 +152,35 @@ private void Activate(Player player)
...
+
             switch (DamageType)
             {
                 default:
                     if (player.Invincible) return;
                     amount *= (float)player.GetLifeResistance(DamageType);
                     iAmount = (int)Math.Round(amount);
                     player.TakeDamage(this, DamageType, amount, Server.Entity.BodyPart.Foot);
+                    pAmount = Math.Abs(iAmount);
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/Hotspot.cs]: iAmount is always positive here, so this statement is effectively pAmount = iAmount, ie. redundant

File: Source/ACE.Server/WorldObjects/Hotspot.cs [hunk 4]
```diff
@@ -152,32 +152,35 @@ private void Activate(Player player)
...
-                    player.UpdateVitalDelta(player.Health, -iAmount);
+                    pAmount = player.UpdateVitalDelta(player.Health, -iAmount);
                     break;
             }
 
             if (!Visibility)
                 EnqueueBroadcast(new GameMessageSound(Guid, Sound.TriggerActivated, 1.0f));
 
-            if (!string.IsNullOrWhiteSpace(ActivationTalk))
+            if (!string.IsNullOrWhiteSpace(ActivationTalk) && pAmount != 0)
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/Hotspot.cs]: all of this code can be nuked, an replaced with 1 update: && iAmount != 0

**Pull request merged**